sdk/documentintelligence/Azure.AI.DocumentIntelligence/tests/DocumentIntelligenceClient/DocumentModelLiveTests.cs

```diff
@@ -34,6 +34,24 @@ public async Task AnalyzeDocumentWithUrlSource()
             ValidateContosoReceiptAnalyzeResult(operation.Value);
         }
 
+        [RecordedTest]
```

Comment:
Is this test bed similar to FR?

Reply:
In general, yes. The overall structure of the client is different but the main scenarios are the same. There are also some additions such as the Base 64 source feature, not present in the previous client.

I also tried to skip most tests I found redundant or not SDK-related to avoid having too many tests as we have in the old client.